ui/app/src/components/workspaces/WorkspaceServices.tsx

```diff
@@ -16,18 +14,11 @@ export const WorkspaceServices: React.FunctionComponent<WorkspaceServicesProps>
 
   return (
     <>
-     <h1>Workspace Services Landing</h1>
-
-      <h2>Services</h2>
-      <ul>
-      {
-        props.workspaceServices.map((ws:WorkspaceService, i:number) => {
-          return (
-            <li key={i}><Link to={ws.id.toString()} onClick={() => props.setWorkspaceService(ws)}>{ws.properties.display_name}</Link></li>
-          )
-        })
-      }
-      </ul>
+     <h1>Workspace Services</h1>
+      <ResourceCardList 
+        resources={props.workspaceServices} 
+        selectResource={(r: Resource) => props.setWorkspaceService(r as WorkspaceService)} 
+        emptyText="This workspace currently has no workspace services."/>
```

Comment:
nit: message uses "currently" which is inconsistent with the workspace services text. Seriously...feel free to ignore this.

Reply:
To make consistent, just remove "currently".
Or...don't :)